src/views/Profile/index.tsx

```diff
@@ -96,7 +78,9 @@ const Profile: React.FC<{ isLoading?: boolean }> = props => {
         }
         catch(e: any) {
             if (e?.data?.error?.name === 'EXPIRED_SIGNATURE') {
-                signAndUpdate(data);
+                const { success } = await handleSignature(signMessage);
```

Comment:
The sign message isn't always the same?
If yes, WDYT about moving the message to the method?

Reply:
keep in mind, we should at some point, have this message on prismic, so we can translate. But will likely be the same.

Reply:
The signMessage is not the message, is the method from utils, that actually signs in the blockchain. It's a hook, i can't call it outside a component and that's the reason why i'm passing it, as a callback to the method.